src/main/java/ch/njol/skript/expressions/ExprArabicNumeral.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * This file is part of Skript.
+ *
+ * Skript is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Skript is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import java.util.HashMap;
+
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+
+@Name("Arabic Numeral")
+@Description("The Arabic Numeral of a Roman Numeral")
+@Examples("set {_r} to arabic numeral of \"XVI\"")
+@Since("INSERT VERSION")
+public class ExprArabicNumeral extends SimplePropertyExpression<String, Number> {
+	
+	private final static HashMap<Character, Integer> map = new HashMap<Character, Integer>() {{
+		put('H', 10000);
```

Comment:
Uh, I just realized, is H an actual roman numeral character? I can't find info on it...

Reply:
@Description:

*Even though the usual maximum roman number is 3999 we added 'H' and 'G' respectively for the 10000 and 5000 numbers, to make the maximum be 39,999.*

Reply:
Otherwise I would've had to put some letters that don't exist in the English alphabet

Reply:
In that case, you should add the description to ExprArabicNumeral as well, since it's only in ExprRomanNumeral

Reply:
Okay